### ISSUE TITLE
GHEvent is missing new events that is breaking Jenkins plugin usage

### ISSUE THREAD

**Issue by philippebailer** [hub4j/github-api#1234]
**Describe the bug**
Getting error:
```
com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `org.kohsuke.github.GHEvent` from String "merge_queue_entry": not one of the values accepted for Enum class: [GOLLUM, ALL, INTEGRATION_INSTALLATION_REPOSITORIES, META, DOWNLOAD, FORK_APPLY, CHECK_RUN, MARKETPLACE_PURCHASE, PROJECT_CARD, PUSH, PAGE_BUILD, TEAM_ADD, REPOSITORY_VULNERABILITY_ALERT, PACKAGE, DELETE, DEPLOYMENT_STATUS, WATCH, FOLLOW, ORG_BLOCK, ORGANIZATION, COMMIT_COMMENT, ISSUE_COMMENT, REPOSITORY_IMPORT, GITHUB_APP_AUTHORIZATION, CODE_SCANNING_ALERT, CHECK_SUITE, FORK, GIST, CONTENT_REFERENCE, REPOSITORY, INSTALLATION_REPOSITORIES, REPOSITORY_DISPATCH, MILESTONE, STAR, LABEL, MEMBERSHIP, SECURITY_ADVISORY, PROJECT_COLUMN, TEAM, PULL_REQUEST_REVIEW, RELEASE, PUBLIC, WORKFLOW_RUN, STATUS, PULL_REQUEST, PROJECT, WORKFLOW_DISPATCH, CREATE, MEMBER, REGISTRY_PACKAGE, INSTALLATION, DEPLOYMENT, PULL_REQUEST_REVIEW_COMMENT, DEPLOY_KEY, PING, ISSUES]
 at [Source: (String)"[{"id":......
```

When trying to connect to github app from jenkins.

**To Reproduce**
1. Set up Jenkins Github App
2. No event subscriptions are needed as this occurs even when nothing is selected
3. Start creating a new Github App Credential in jenkins 
4. Once all the fields are set hit `Test` to test the connection 
5. It should fail with the above error

**Expected behavior**
Jenkins Github App Connectivity should work with new events

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser chrome
 - Version latest

**Additional context**
Create this PR with possible fix and more info https://github.com/hub4j/github-api/pull/1233

Creating this issue as I was unsure if the PR was enough and also for visibility

**Comment by philippebailer** [comment]
It seems that release https://github.com/hub4j/github-api/releases/tag/github-api-1.128 may have actually introduce UNKNOWN GHEvent enum handling but the jenkins github-api-plugin is at version 122

My PR might not be needed if release 1.128 was meant to resolve this issue

**Comment by philippebailer** [comment]
@bitwiseman considering the above I'm not sure what I need to be doing to resolve this issue.

What is the process for releasing the jenkins github-api-plugin (https://github.com/jenkinsci/github-api-plugin)?
Is this something that can be done quickly or do you have any other trouble shooting suggestions?

**Comment by philippebailer** [comment]
Ok that's great hopefully this resolves our issue.

I'll keep an eye on this and test it as soon as its live.

Thanks @bitwiseman

**Comment by philippebailer** [comment]
@bitwiseman saw that the PR was merged, is the release in progress? I don't see a new tag yet in https://github.com/jenkinsci/github-api-plugin

**Comment by bitwiseman** [comment]
> @bitwiseman saw that the PR was merged, is the release in progress? I don't see a new tag yet in https://github.com/jenkinsci/github-api-plugin

I'm on leave right now, working on this in my spare time (from my phone right now). I'll get to it today.

**Comment by philippebailer** [comment]
Sorry for the trouble and thank you for keeping an eye on this

**Comment by bitwiseman** [comment]
I've released github-api-plugin 1.133. That should unblock you.  
I've also merged your PR.  It will be in 1.134 .  Thanks for contributing!

**Comment by philippebailer** [comment]
Yes its working and we are unblocked! Thanks @bitwiseman for helping us get through this quickly and closing my PR
